lib/cmr/cmr_client.rb

```diff
@@ -306,6 +316,19 @@ def delete_subscription(provider_id, native_id, token)
       delete(url, {}, nil, headers.merge(token_header(token)))
     end
 
+    # MMT does not need to ingest granules in any environment that is not dev or
+    # test.
+    def ingest_granule(metadata, provider_id, native_id)
+      url = "http://localhost:3002/providers/#{provider_id}/granules/#{encode_if_needed(native_id)}"
```

Comment:
I think it might be better to add a guard clause here: `return unless Rails.env.development? || Rails.env.test?`